pkg/controllers/routercerts/controller.go

```diff
@@ -186,10 +180,9 @@ func (c *routerCertsDomainValidationController) validateRouterCertificates() ope
 		klog.Infof("system cert pool not available: %v", err)
 		verifyOptions.Roots = x509.NewCertPool()
 	}
-	if !common.IsDefaultSecret(secret) {
+	if !isDefault {
```

Comment:
NOTE to me: I still need to think about this.

Reply:
yeah, remove this, it's effectively a no-op and so this controller should be a no-op when custom route certificate is supplied. It was designed to check the validity of the router-certs secret, the route's TLS validity is checked in the other endpointcheckers anyway.